tensorflow/examples/tutorials/mnist/fully_connected_feed.py

```diff
@@ -251,9 +251,9 @@ def main(_):
       help='Batch size.  Must divide evenly into the dataset sizes.'
   )
   parser.add_argument(
-      '--train_dir',
+      '--data_dir',
```

Comment:
We probably want to keep those separate. It could be an issue with naming, for instance,
output_dir
input_data_dir
?
(open to ideas -- but generally, I think it's a bad idea to mix input and output in the same tree)